core/tests/protractor_utils/AdminPage.js

```diff
@@ -272,6 +287,50 @@ var AdminPage = function() {
       expect(name).toEqual(expectedUsernamesArray[j]);
     }
   };
+
+  this.addReviewer = async function(name, newRole) {
+    await waitFor.elementToBeClickable(
+      adminRolesTab, 'Admin Roles tab is not clickable');
+    await adminRolesTab.click();
```

Comment:
FYI Sandeep created an awesome [`actions.js`](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/action.js) file with utilities that take care of the waiting when clicking, sending keys, and more. That might be easier than writing all the waitFor statements manually like you have here